tests/duplicate-instance.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test } from "./fixtures/lxd-test";
+import { deleteAllImages } from "./helpers/images";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitInstance,
+} from "./helpers/instances";
+
+test("Duplicate an instance", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+
+  await page.goto("/ui/");
+  await visitInstance(page, instance);
```

Comment:
```suggestion
  await visitInstance(page, instance);
```
We can use a shared instance from the instance suite.